### ISSUE TITLE
Window position on screen

### ISSUE THREAD

**Issue by dbzakj** [celluloid-player/celluloid#892]
Is there a way to have the program remember the window position on screen? It seems to remember the window size, but always starts several pixels away from the left edge of the screen. This happens on both Linux Mint and Arch (XFCE4), using gnome-mpv and the latest version from pacman respectively.

**Comment by gnome-mpv** [comment]
No. On Wayland, apps can't set their own positions, so there's nothing we can do there. On X11, it's still possible but  requires some extra work.

**Comment by gnome-mpv** [comment]
We'll have to implement a mechanism for it in Celluloid. It can't be done with e.g. a config file.

**Comment by dbzakj** [comment]
Ok, just a feature request then. Thanks.

**Comment by dbzakj** [comment]
Oddly enough, I followed these steps, just to have no border, and it latches to the edge of the screen, thereby solving my request in an unintentional way.

https://github.com/celluloid-player/celluloid/issues/597

https://i.imgur.com/7TNndRU.jpg

https://i.imgur.com/s80r4ee.jpg
